addons/mail/static/src/owl/store/messaging_actions.js

```diff
@@ -468,6 +468,26 @@ const actions = {
         env.call('bus_service', 'startPolling');
         dispatch('_startLoopFetchPartnerImStatus');
     },
+    /**
+     *
+     * @param unused
```

Comment:
Doc to be adapted: missing description text at the top, and here param0.state is actually used

Reply:
Done